### DISCUSSION TITLE
Audio recording

### DISCUSSION THREAD

**Discussion by ghost** [discord-jda/JDA#1841] [Questions and Help]
Good evening everyone,
Discord itself does not support recording conversations by default, but it is possible to intercept conversations. That this works has already proven a friend, but he does not use the JDA. Since the JDA nevertheless also offers the possibility to intercept sounds, we come to my question: How is it possible to record a conversation and convert it into an .mp3 file that you save somewhere on your hard drive? I have already looked at some examples, but it came to no result and I discarded my programming code. Since I think there are many who can do this better than me, I hope for some help.

Kind regards
Nicklas

**Comment by jan-tennert** [comment]
This might help:
https://stackoverflow.com/questions/56003861/pcm-audio-to-file
